presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java

```diff
@@ -116,6 +120,11 @@
             .toFormatter()
             .withOffsetParsed();
 
+    // Before 1900, Java Time and Joda Time are not consistent with java.sql.Date and java.util.Calendar
+    // Since January 1, 1900 UTC is still December 31, 1899 in other zones, we are adding a 1 year margin.
+    private static final int MODERN_ERA_YEAR = 1901;
```

Comment:
No need for this constant, since it's only used in the constant below. It would actually be easier to read the below if it is simply said `1901, 1, 1` as then it's easy to compare with the comment above.